docs/Installation/Installation_Guide.rst

```diff
@@ -346,21 +332,17 @@ Cluster Services
 Services to operate and monitor your MetalK8s cluster are provided. To access
 these dashboards:
 
-1. Make sure kubectl is installed on your local machine::
-
-   $ yum install kubectl
-
-2. Copy the credentials in
+1. Copy the credentials in

```

Comment:
Would be helpful to specify where to save this file in local machine (`~/.kube/config` for ubuntu as well as macos, I believe)

Reply:
After discussing with William - asking to copy the file to 'your local machine' may be misleading, as 'local machine' is often associated with your computer device. This step can be simplified to just specifying that we export `KUBECONFIG` in our deployment node, or something similar.

Reply:
done.